src/main/groovy/com/blackbaud/templates/GitRepo.groovy

```diff
@@ -52,11 +60,17 @@ class GitRepo {
         open(repoDir)
     }
 
-    static GitRepo initGitRepo(File repoDir) {
+    static GitRepo initGitHubRepo(String name, File repoDir) {
```

Comment:
this looks like the same code as below, perhaps something like...
```
initRepo(String repoUrl, File repoDir) {
        repoDir.mkdirs()
        GitRepo git = init(repoDir)
        git.setRemoteUrl("origin", repoUrl)
        git
```
and then construct the actual url in the initGitHubRepo/initVstsGitRepo methods